versioned/mongodb/src/main/java/com/dremio/nessie/versioned/store/mongodb/CodecProvider.java

```diff
@@ -61,7 +61,7 @@ void write(BsonWriter writer, Map<String, Entity> attributes) {
      * @param field the name of the field as represented in BSON
      * @param value the entity that will be serialized.
      */
-    private void writeField(BsonWriter writer, String field, Entity value) {
+    void writeField(BsonWriter writer, String field, Entity value) {
```

Comment:
I must assume this code now needs to be accessed from outside the class. Is that the objective?